FreeRTOS-Plus/Demo/FreeRTOS_Plus_TCP_Echo_Posix/Makefile

```diff
@@ -52,31 +52,10 @@ SOURCE_FILES += ${FREERTOS_DIR}/Source/portable/MemMang/heap_3.c
 
 
 # FreeRTOS TCP
-SOURCE_FILES += ${FREERTOS_PLUS_DIR}/Source/FreeRTOS-Plus-TCP/source/FreeRTOS_ARP.c
```

Comment:
These makefile changes are added for only 2 demos? Any reason?

Reply:
I believe all the other +TCP demos are not based on Makefiles